### DISCUSSION TITLE
Laravel User Monitoring

### DISCUSSION THREAD

**Discussion by milwad-dev** [HassanZahirnia/laravel-package-ocean#205] [Package Suggestions]
Please add Laravel-User-Monitoring from binafy.

https://github.com/binafy/laravel-user-monitoring

**Comment by HassanZahirnia** [comment]
Hey! This package also already exists in the database!

Please consider searching in the website before suggesting a new package, thanks!

**Comment by milwad-dev** [comment]
Hhh, sorry
